pkg/virt-api/webhooks/validating-webhook/admitters/pod-eviction-admitter.go

```diff
@@ -40,7 +40,7 @@ func (admitter *PodEvictionAdmitter) Admit(ar *admissionv1.AdmissionReview) *adm
 		return validating_webhooks.NewPassingAdmissionResponse()
 	}
 
-	if !isVirtLauncher(pod) {
```

Comment:
The intention behind this condition is approving eviction requests on pods that aren't `virt-launcher`s.

Please consider adding another condition that will check if the pod is completed:
```suggestion
	if !isVirtLauncher(pod) {
		return validating_webhooks.NewPassingAdmissionResponse()
	}
	
	// No need to trigger evacuation of a VMI who's virt-launcher pod is completed
	if isCompleted(pod) {
	    return validating_webhooks.NewPassingAdmissionResponse()
	}
```

It makes the flow clearer and more explicit IMO.

Reply:
Another (more compact) alternative:
```suggestion
	if !isVirtLauncher(pod) || isCompleted(pod) {
		return validating_webhooks.NewPassingAdmissionResponse()
	}
```